scripts/run_backend_tests.py

```diff
@@ -90,8 +89,19 @@
     os.path.join(common.THIRD_PARTY_DIR, 'webencodings-0.5.1'),
 ]
 
-COVERAGE_PATH = os.path.join(
-    os.getcwd(), '..', 'oppia_tools', 'coverage-4.5.4', 'coverage')
+# Explicitly pass all current environment variables into subprocess.
+# Otherwise, there will be dll error for running coverage. (For Windows)
+# pylint: disable=unicode-builtin
+SUBPROCESS_ENV = {
```

Comment:
Sorry, I forgot to point out, this one is unnecessary now.

Reply:
Removed